test/unit/util.test.js

```diff
@@ -333,6 +333,53 @@ describe('UTILITY FUNCTION TESTS ', function () {
       expect(collectionVariables).to.have.key('petUrl');
       done();
     });
+
+    // https://github.com/postmanlabs/openapi-to-postman/issues/80
+    it('should generate trie taking swagger specs with root endpoint declaration as input.', function (done) {
+      var openapi = {
+          'openapi': '3.0.0',
+          'info': {
+            'version': '1.0.0',
+            'title': 'Swagger Petstore',
+            'license': {
+              'name': 'MIT'
+            }
+          },
+          'servers': [
+            {
+              'url': 'http://petstore.swagger.io/{v1}'
+            }
+          ],
+          'paths': {
+            '/': {
+              'get': {
+                'summary': 'List all pets',
+                'operationId': 'listPets',
+                'responses': {
+                  '200': {
+                    'description': 'An paged array of pets',
+                    'content': {
```

Comment:
The content section can be removed - not necessary to what's being tested, and adds unnecessary bulk to the test code.

Reply:
fixed